publish.html

```diff
@@ -0,0 +1,580 @@
+---
+layout: default
+title: Publishing with OMERO
+menu-id: l1-workflow
+pdfs:
+    - publish.pdf
+description: How to create figures and movies from data on OMERO, and publically share images.
+---
+<div class="page-navigation">
+<h2>Contents</h2>
+<p>
+<a href="#introduction">Introduction</a><br />
+<a href="#figures">Creating figures</a><br />
+<a href="#public">Using a public group</a><br />
+
+</p>
+</div><!-- End Page Navigation -->
+
+<div class="line-block">
+<div class="line"><br /></div>
+</div>
+
+<h2><a class="anchor" id="introduction"></a>Introduction</h2>
+
+<p>The principal tool for creating figures using images on OMERO is OMERO.figure. This is an optional installation, so may not be available on your server. administrator. Details can be found on the <a href="http://help.openmicroscopy.org/figure.html" target="_blank" class="external-url">OMERO.figure</a> page. If you would like to use OMERO.figure, speak to your OMERO system administrator.</p>
```

Comment:
'may not be available on your server. administrator. Details…'

I don't think you want this 'administrator' here - it's in the pdf too.